### PR TITLE
dates and times initial

### PR DESCRIPTION
**Pull request by richardjtelford**
Modified from tutorial. Is anything missing?

### CODE REVIEW

File: data_manipulation/dates_times.Rmd [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+  mutate(hour = hour(date_time)) %>% 
+  group_by(hour) %>% 
+  summarise(mean_temperature = mean(Lufttemperatur))
+```
+
+
+## Plotting dates
+
+We can plot the temperature data with `ggplot`.
+
```
Comment (audhalbritter) [inline data_manipulation/dates_times.Rmd:301]: plot temperature over time

File: data_manipulation/dates_times.Rmd [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+```{r}
+ggplot(temperature2, aes(x = date_time, y = Lufttemperatur)) + 
+  geom_line() +
+  scale_x_datetime(date_labels = "%A %d %B") #see also date_breaks to change label positions
+```
+
+
+We can also plot times. 
+
```
Comment (audhalbritter) [inline data_manipulation/dates_times.Rmd:320]: not sure what you mean with times? Everything is about times here

**Review by audhalbritter — COMMENTED** [review]
Looks good, I had a few minor comments.

File: data_manipulation/dates_times.Rmd [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+The default timezone for the `lubridate` functions is UTC (approximately Greenwich Mean Time).
+This is normally fine unless you are dealing with local times in multiple time-zones, or because you need to allow for daylight saving time.
+You can change the time zone with the argument "tz".
+
+```{r timezone}
+mdy_hm("July 1 2001 2:14", tz = "Europe/Oslo")
+```
+
+The function `OlsonNames()` returns a vector of valid time zones.
+See the help file for details.
```
[the remark this answers is not in the document]
Comment (richardjtelford) [reply data_manipulation/dates_times.Rmd:165]: There is a lot of technical information in that helpfile, but little that is useful for actually using it on a day-to-day basis. Changed this sentence to one describing the output of the chunk

File: data_manipulation/dates_times.Rmd [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+
+## Dates and `dplyr`
+
+As an ecologist, I spend a lot of time processing climate data  for comparison with ecological data.
+The data are typically in R in a data.frame, so `dplyr` is useful for manipulating them.
+This section shows how to use `lubridate` together with `dplyr`.
+
+The dataset `temperature` includes a few days of air temperature from the [GFI weather station](https://veret.gfi.uib.no/), Bergen.
+
+The dataset has three columns, `Dato`, `Tid` and `Lufttemperatur`.
```
[the remark this answers is not in the document]
Comment (richardjtelford) [reply data_manipulation/dates_times.Rmd:237]: I think it is fine - anyone can guess what these are.

File: data_manipulation/dates_times.Rmd [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+
+```{r print-temperature}
+temperature
+```
+
+### Using `mutate` to make a new column
+
+I want to make a single date-time column. 
+To do this I need to combine the data and time columns with `paste`, and then use a `lubridate` function for the conversion. 
+
```
[the remark this answers is not in the document]
Comment (richardjtelford) [reply data_manipulation/dates_times.Rmd:249]: OK

File: data_manipulation/dates_times.Rmd (hunk 1 above)
Comment (richardjtelford) [reply data_manipulation/dates_times.Rmd:301]: ok

File: data_manipulation/dates_times.Rmd (hunk 2 above)
Comment (richardjtelford) [reply data_manipulation/dates_times.Rmd:320]: bare times without dates. 15:34. with hms class. Clarified

File: data_manipulation/dates_times.Rmd [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+## Dates and `dplyr`
+
+As an ecologist, I spend a lot of time processing climate data  for comparison with ecological data.
+The data are typically in R in a data.frame, so `dplyr` is useful for manipulating them.
+This section shows how to use `lubridate` together with `dplyr`.
+
+The dataset `temperature` includes a few days of air temperature from the [GFI weather station](https://veret.gfi.uib.no/), Bergen.
+
+The dataset has three columns, `Dato`, `Tid` and `Lufttemperatur`. 
+I have imported the data with `readr::read_csv` so the first two columns have automatically been converted to date and time formats.
```
[the remark this answers is not in the document]
Comment (richardjtelford) [reply data_manipulation/dates_times.Rmd:238]: ok

File: data_manipulation/dates_times.Rmd [hunk 7]
```diff
@@ -0,0 +1,335 @@
...
+
+The format for dates with times is `POSIXct`.
+Internally, this is the number of seconds since 1970-01-01.
+
+Sometimes it is useful to have a time-only format, this is provided by the `hms` package.
+
+## Converting dates and times
+
+This can be done using base R.
+
```
[the remark this answers is not in the document]
Comment (richardjtelford) [reply data_manipulation/dates_times.Rmd:88]: ok

**Pull request merged**